docs/sql-ref-syntax-ddl-create-table.md

```diff
@@ -20,10 +20,10 @@ license: |
 ---
 
 ### Description
-`CREATE TABLE` statement is used to define a table in an exsisting database. 
+`CREATE TABLE` statement is used to define a table in an existing database. 
 
 The CREATE statements:
-* [CREATE TABLE USING DATASOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
+* [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
```

Comment:
nit: `DATA_SOURCE` is preferred against `DATASOURCE`? It seems the current codebase has both formats; https://gist.github.com/maropu/a7f780208ff50c990a52121378531c25

Reply:
I am confused time to time if it's Datasource or Data source :-).

Reply:
Yea, if we can, better to use one of them consistently.

Reply:
English-wise, "data" and "source" are two words...

Reply:
Let's go with `DATA_SOURCE` in public documents, we can fix the internal naming later.